src/js/app.js

```diff
@@ -11,128 +17,171 @@ require('./search-box');
 require('./header-nav');
 require('./expandables');
 
-window.censusAPI = {};
+window.callbacks = {};
+
+callbacks.censusAPI = function(data, rural) {
+  if (addr.isFound(data.result)) {
+    var result = {};
+    result.x = data.result.addressMatches[0].coordinates.x;
+    result.y = data.result.addressMatches[0].coordinates.y;
+
+    $.when(tiger(result.x, result.y, '86'),
+      tiger(result.x, result.y, '66'),
+      tiger(result.x, result.y, '64'))
+    .then(function(countyData, UCData, UAData) {
+      var censusCounty = JSON.parse(countyData[0]);
+      var censusUC = JSON.parse(UCData[0]);
+      var censusUA = JSON.parse(UAData[0]);
+
+      result.input = data.result.input.address.address;
+      result.address = data.result.addressMatches[0].matchedAddress;
+      result.countyName = censusCounty.features[0].attributes.BASENAME;
+
+      var fips = censusCounty.features[0].attributes.STATE + censusCounty.features[0].attributes.COUNTY;
+
+      if(addr.isInCounty(fips, rural)) {
+        result.rural = 'Yes';
+        result.type = 'rural';
+      } else {
+        if(addr.isUrban(censusUC.features, censusUA.features)) {
+          result.rural = 'Yes';
```

Comment:
ahhh that makes more sense. Is '-' the default? That could also go in the renderer. Like if result.rural isn't set (so undefined rather than null), put in '-', else convert the boolean. But don't do that if you can ever transition from 'Yes'/'No' to empty.

Honestly this is probably fine either way ^_^

hohoho